cdap-kubernetes/src/main/java/io/cdap/cdap/master/environment/k8s/KubeMasterEnvironment.java

```diff
@@ -1087,4 +1100,22 @@ void setConnectTimeout(int connectTimeoutSec) {
   void setReadTimeout(int readTimeoutSec) {
     this.readTimeoutSec = readTimeoutSec;
   }
+
+  @VisibleForTesting
+  Map<String, String> parseLoadBalancerAnnotations(@Nullable String configValue) {
+    Map<String, String> map = new HashMap<>();
+    if (Strings.isNullOrEmpty(configValue)) {
+      return map;
+    }
+    for (String property : configValue.split(",")) {
```

Comment:
Use the guava `Splitter.on(',').withKeyValueSeparator('=')` instead.

Reply:
Thanks for sharing, I wasn't aware of this utility function.